### PR TITLE
80 character rule nit fixes

### PR DESCRIPTION
**Pull request by mlunoe**
---
Follow up PR to this one: https://github.com/dcos/dcos-ui/pull/1097

---

This just fixes some 80 character rules

### CODE REVIEW

**Comment by nLight** [comment]
Thanks @mlunoe! 👍  Shall we enable warnings for that rule?

File: src/js/components/CheckboxTable.js [hunk 1]
```diff
@@ -79,22 +79,27 @@ class CheckboxTable extends React.Component {
...
+      data
+    } = this.props;
 
     if (!allowMultipleSelect) {
       return null;
     }
 
-    let checkedItemsCount = Object.keys(checkedItemsMap).length;
-    let disabledItemsCount = Object.keys(disabledItemsMap).length;
+    let checkedCount = Object.keys(checkedItemsMap).length;
```
Comment (orlandohohmeier) [inline src/js/components/CheckboxTable.js]: I'm ok with this change but don't think that shortening variables names is generally a good fix to obey to the 80 characters rule.

**Comment by orlandohohmeier** [comment]
LGTM! 🚢

File: src/js/components/CheckboxTable.js (hunk 1 above)
Comment (mlunoe) [inline src/js/components/CheckboxTable.js]: I initially moved code around, until I realized I thought the `Items` part was redundant. And so I landed on this. But yes, I do agree. That should not be a fix

**Comment by mlunoe** [comment]
@orlandohohmeier and I have been working on a solution for that. There was some trouble about how we would tackle our exceptions, or how, if we make it a hard rule, we would handle breaking up our long canvas class names (and other situations)

**Pull request merged**